content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/title.html

```diff
@@ -22,6 +22,8 @@
      class="cmp-title">
     <h1 class="cmp-title__text" data-sly-element="${title.type}"><a
             data-sly-unwrap="${!title.linkURL || title.linkDisabled}" class="cmp-title__link" href="${title.linkURL}"
+            aria-label="${title.linkAccessibilityLabel}"
```

Comment:
If we make the `Title` return a `Link` then we can either do: `aria-label="title.link.htmlAttributes['aria-label']"` or just batch add all like `data-sly-attribute="${title.link.htmlAttributes}"`